packages/math-input/src/index.ts

```diff
@@ -9,5 +9,5 @@ export {default as KeypadInput} from "./components/input/math-input";
 export {keypadElementPropType} from "./components/prop-types";
 export {default as Keypad} from "./components/provided-keypad";
 export {default as KeyConfigs} from "./data/key-configs";
-export {default as Keys} from "./data/keys";
+export {default as Key} from "./data/keys";
```

Comment:
Can you do a quick check in webapp and mobile to see if this type is used?  If so, we'll need to be careful as we roll out this change to update the usages.

Reply:
Oh, I didn't know we made these types available externally. If that's the case, this is more than a patch change!

Reply:
Since it's exported from the root `index.ts` it's externally available, so let's just check.  It's totally fine to make this change a `major`... version numbers are free. :)